sdk/identity/Azure.Identity/tests/samples/BreakingChangesSnippets.cs

```diff
@@ -14,5 +14,25 @@ public void SetExcludeSharedTokenCacheCredentialToFalse()
             });
             #endregion
         }
+
+        public void AddExplicitAdditionallyAllowedTenants()
+        {
+            #region Snippet:Identity_BreakingChanges_AddExplicitAdditionallyAllowedTenants
+            var credential = new DefaultAzureCredential(new DefaultAzureCredentialOptions
+            {
+                AdditionallyAllowedTenants = { "0000-0000-0000-0000", "1111-1111-1111-1111" }
```

Comment:
Noted above, but just in case: seems these should be valid GUIDs. If too long, just put them on separate lines.